Makefile

```diff
@@ -1,9 +1,3 @@
-models:
-	docker-compose up -d db
-	echo "Waiting for db to be ready..."
-	sleep 5
-	PGSSLMODE=disable xo postgresql://postgres:toor@localhost:5432/bcda -o models
```

Comment:
Now that `xo` has been completely removed from Makefile, does the `models` target need to be replaced with something that is GORM-specific?  Will we need to continue to execute the model generation in the future, or is this now completely obsolete?

Reply:
completely obsolete.  GORM doesn't write go code the way XO did.  It handles dynamic query execution at run time instead of prewriting it all.